packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/option/index.css

```diff
@@ -0,0 +1,59 @@
+
+.face-value-option {
+  display: inline-grid;
+  grid-template-columns: auto 1fr;
+  align-items: center;
+  column-gap: var(--boxel-sp-xl);
+  padding: var(--boxel-sp-lg) var(--boxel-sp);
+  border-radius: var(--boxel-border-radius);
+  box-shadow: 0 0 0 1px var(--boxel-light-400);
+  transition: box-shadow var(--boxel-transition);
+}
+
+.face-value-option:hover {
+  box-shadow: 0 0 0 1px var(--boxel-dark);
+}
+
+.face-value-option--checked,
+.face-value-option:focus,
+.face-value-option:focus-within {
+  box-shadow: 0 0 0 2px var(--boxel-highlight);
+  outline: 1px solid transparent;
+}
+
+.face-value-option__input {
+  appearance: none;
```

Comment:
Safari needs this prefixed with `-webkit-`: 

https://developer.mozilla.org/en-US/docs/Web/CSS/appearance

![image](https://user-images.githubusercontent.com/39579264/123088846-d3c44d00-d458-11eb-83ad-e62f201adedc.png)